### PR TITLE
DO NOT MERGE: Test new CODEOWNERS file

### PR DESCRIPTION
**Pull request by negz**
### Description of your changes

I'm opening this to test https://github.com/crossplane/crossplane/pull/3229. I would expect this PR to have @hasheddan and two members of @crossplane/crossplane-reviewers assigned to it. I would _not_ expect the PR to be mergeable until Dan (or another maintainer) approved it.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

This code IS the test!

### CODE REVIEW

**Comment by negz** [comment]
It looks like assignment worked as expected. Now I'd like to confirm that approval from a member of @crossplane/crossplane-reviewers is non-binding (i.e. that the PR still needs @crossplane/crossplane-maintainers approval before it can be merged).

**Review by tnthornton — APPROVED** [review]

**Comment by negz** [comment]
Thanks for helping test folks! It seems like this works almost exactly as I'd have liked. @tnthornton can approve the PR such that a maintainer can merge it, but as a member of @crossplane/crossplane-reviewers can't merge it himself.

**Pull request closed without merging**
